src/test/bin/quic_gtest.cpp

```diff
@@ -113,7 +113,9 @@ class QuicTestEnvironment : public ::testing::Environment {
 #if defined(QUIC_API_ENABLE_PREVIEW_FEATURES)
             if (UseQTIP) {
                 Config.PollingIdleTimeoutUs = 10000;
-                Config.Flags |= QUIC_EXECUTION_CONFIG_FLAG_QTIP;
+                MsQuicSettings Settings;
+                Settings.SetQtipEnabled(TRUE);
```

Comment:
```suggestion
                Settings.SetQtipEnabled(true);
```
C++ code uses `true` instead of `TRUE`.

Reply:
Done.